### ISSUE TITLE
Ingest 100,00 brightest points and overlay them on both views of the brain

### ISSUE THREAD

**Issue by mlee156** [NeuroDataDesign/seelviz-f16s17#46]
Ingest 100,00 brightest points and overlay them on both views of the brain

**Comment by toekneesunshine** [comment]
The process for doing this is in an ipynb documented [here](https://github.com/alee156/cldev/blob/master/Tony/ipynb/Generating%2BBlocks%2Band%2BUploading%2BPoints.ipynb).

This is the way Greg showed me -- effectively, we take the original 3D array and then:  
1.  Duplicate the inImg
2.  Set all values to zero by directly editing each element of the array  
3.  For all the bright points read from our CSV, we generate a list of "cross" shaped object (eg: for each bright point we find the spot with +x, +y, +z, -x, -y, -z, thereby creating a plus shaped 3D shape for each original bright point).
3.  Set all values of interest (eg: bright points) to either one specific brightness value or alternative brightness values.  After consultation with Greg, I used a counter that just set different colors for each thing.

**Comment by toekneesunshine** [comment]
After running through the main body of code [here](https://github.com/alee156/cldev/blob/master/Tony/ipynb/Generating%2BBlocks%2Band%2BUploading%2BPoints.ipynb), (see issue [here](https://github.com/NeuroDataDesign/seelviz/issues/46)), we upload as an annotation channel (as recommended by Greg during my brief meeting w/ him today).

Technically, something should show up in ndviz -- however, nothing really shows up despite the upload completing.

I first tried thrice uploading as an annotation channel (each point w/ ID 120, 255, and then 700).  I then tried as an annotation channel with different values (eg: each point has a unique non-zero ID).  I also tried uploading as a uint8 type image.  In all cases, I couldn't tell if anything was showing up in ndviz.

To confirm that it wasn't the small size of the "cross", I also uploaded "bigBox", where I generated a very large box of 50 x 50 x 50 at the origin.  I repeated all the uploads, but to no avail.

By checking the indices I edited, I do see that there are non-zero values, so it must be in how I'm uploading the ITK images.
